tests/test_processor.py

```diff
@@ -5,8 +5,8 @@
 import logging
 import shutil
 import git
-import flexmock
 
+from flexmock import flexmock
```

Comment:
Is this needed? I see `flexmock-0.10.4` used in the tests and [according to the docs](https://flexmock.readthedocs.io/en/latest/user-guide/?highlight=__init__#setup) `import flexmock` is enough since `0.10.0`.

Reply:
If you tell me how to explain it to PyCharm (to not warn 30 times that "'flexmock' is not callable") then it's not (needed). :-)